code_generator_TopCPToolkit/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,58 @@
+name: CI/CD
```

Comment:
Oops - we need to delete all of the service level workflows. This is all now handled at the repo level [deploy-config.json](https://github.com/ssl-hep/ServiceX/blob/develop/.github/workflows/deploy-config.json)

Reply:
@BenGalewsky - I've added TopCP to the `deploy-config.json` to be included in the CI/CD. And we can delete all `ci.yaml` under each service like [code_generator_funcadl_uproot/.github/workflows/ci.yaml](https://github.com/ssl-hep/ServiceX/blob/develop/code_generator_funcadl_uproot/.github/workflows/ci.yaml)?